docs/content/concepts/logging/python-logging.mdx

```diff
@@ -5,13 +5,71 @@ description: Dagster is compatible and configurable with Python's logging module
 
 # Python Logging
 
-Dagster is compatible and configurable with [Python's logging module](https://docs.python.org/3/library/logging.html). You can specify and configure handlers and formatters to create custom log messages.
+Dagster is compatible and configurable with [Python's logging module](https://docs.python.org/3/library/logging.html). At the moment, you can access the configuration options through your `dagster.yaml` file, which will apply the contained settings to any run launched from your instance. These settings include which python loggers you'd like to capture from, what log level to set your loggers to, and which handlers / formatters you'd like to use to process log messages produced from your runs.
 
-## Configuring Python Logging <Experimental />
+## Capturing Python Logs <Experimental />
 
-In your config YAML, you can configure handlers and formatters that will apply to the Dagster instance, so all pipeline runs will share the same logging configuration.
+By default, logs generated using the python logging module are not captured into the Dagster ecosystem. This means that they are not stored in the Dagster event log, will not be associated with any Dagster metadata (such as step key, run id, etc.), and will not show up in the default view of Dagit.
 
-```yaml file=/concepts/logging/python_logging_config.yaml
+For example, imagine you have the following code:
+
+```python file=/concepts/logging/python_logger.py startafter=start_python_logger endbefore=end_python_logger dedent=4
+import logging
+from dagster import solid
+
+my_logger = logging.getLogger("my_logger")
+
+@solid
+def ambitious_solid():
+
+    try:
+        x = 1 / 0
+        return x
+    except ZeroDivisionError:
+        my_logger.error("Couldn't divide by zero!")
+
+    return None
+```
+
+With the default behavior, because this code uses a custom python logger (instead of `context.log`), this log statement will not be added as an event in the Dagster event log, and therefore won't show up in Dagit. However, sometimes it is desirable to change this default behavior, and treat these sorts of log statements identically to how Dagster treats `context.log` calls.
+
+This can be accomplished by setting the `managed_python_loggers` key in your dagster.yaml file to a list of python logger names that you would like to capture.
```

Comment:
It would be helpful to include an example yaml file showing how you'd capture logs from specific loggers